quick-start/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+docker run -p 8200:8200 -e 'VAULT_DEV_ROOT_TOKEN_ID=dev-only-token' vault
```

Comment:
Would it make sense to include go run here as well to make the script mirror what we have in the sample app (run.sh starts both the environment & the app)

Reply:
I'd also add have `docker stop` at the end and `--rm` for the docker run

Reply:
Yup, good points